dbft.go

```diff
@@ -168,22 +168,22 @@ func (d *DBFT[H]) OnTransaction(tx Transaction[H]) {
 }
 
 // OnTimeout advances state machine as if timeout was fired.
-func (d *DBFT[H]) OnTimeout(hv HV) {
+func (d *DBFT[H]) OnTimeout(height uint32, view byte) {
```

Comment:
I also had this thought but for some reason I thought that you'll be against such changes. This way is simpler and LGTM.